Automattic/wp-calypso
# Update/add monthly breakdown to composite checkout line items

#### Changes proposed in this Pull Request

* Add a monthly breakdown to plan line items in composite checkout. Depends on D44051-code

To do:
- [x] Needs design fixup

#### Testing instructions

Enter checkout with a plan; the line item should include an (accurate!) monthly breakdown of the plan cost before coupons or discounts. The monthly price should match that shown on the plans page. Check that the monthly price updates when changing the plan length.

![Screen Shot 2020-05-27 at 1 57 45 PM](https://user-images.githubusercontent.com/9310939/83061338-6e6a1680-a022-11ea-8838-9ed80f4398f2.png)
